lib/reform/form/active_model/validations.rb

```diff
@@ -233,6 +234,7 @@ def initialize(form)
         def method_missing(m, *args, &block)
           __getobj__.send(m, *args, &block) # send all methods to the form, even privates.
         end
+        ruby2_keywords(:method_missing) if respond_to?(:ruby2_keywords, true)
```

Comment:
Is `#ruby2_keywords` only defined in Ruby >= 3.0?

Reply:
It's defined since Ruby 2.7 in which the keyword argument changes were added. I actually just copied the example from the ruby docs (https://rubyapi.org/2.7/o/module#method-i-ruby2_keywords).